plugin/core/windows.py

```diff
@@ -540,11 +541,33 @@ def discard(self, window: sublime.Window) -> None:
             wm.destroy()
 
 
+class RequestTimeTracker:
+    def __init__(self) -> None:
+        self._start_times = {}  # type: Dict[int, float]
+
+    def start_tracking(self, request_id: int) -> None:
+        self._start_times[request_id] = time()
```

Comment:
`time.time()` in general has only a precision of 1 second. So it's not very usefult to use it to measure a duration and show it in milliseconds.
I think you need [`time.perf_counter()`](https://docs.python.org/release/3.3.6/library/time.html?highlight=time#time.perf_counter) instead.

Reply:
It certainly has higher precision on mac OS as shown in the screenshot. :)
But `perf_counter` does sounds a more appropriate one, thanks.